### ISSUE TITLE
Change Package Visibility in Settings UI

### ISSUE THREAD

**Issue by jtimberman** [habitat-sh/builder#1042]
### Describe the Enhancement:

I would like to change a package's visibility settings without connecting a plan file, either through the Builder UI while logged in, or via a `hab` CLI.

### Describe the Need:

As a user of the Builder UI, I would like to make a package private (or public) without connecting a plan file because I have another process for building and uploading packages to Builder. For example, I could be doing a proof of concept and do my build/upload from a development machine. Another example is doing a build and upload from a CI/CD system. As it currently works, I have to connect a plan to a repository using the UI (which can be slow due to pagination on, e.g., GitHub, or difficult due to a large number of repositories in the list). 

### Current Alternative

I can currently use a `curl` command against the API like this:

```
curl -v \
-X PATCH \
-H "Accept: application/json" \
-H "Authorization: Bearer $HAB_AUTH_TOKEN" \
https://bldr.habitat.sh/v1/depot/pkgs/ORIGIN/PACKAGE/VERSION/TIMESTAMP/private
```

### Can We Help You Implement This?:

Yes please. I don't have cycles to implement this feature.

**Comment by jtimberman** [comment]
This is related to #1043 :)

**Comment by apriofrost** [comment]
### Solution V1
[View Mockup] (https://chef.invisionapp.com/share/BETDOWL5RYA#/378176306_pkg_Visibility_Settings-V1)

### Acceptance Criteria
- [ ] Change the section title under “origin settings” from “default package settings” to “default package visibility settings”
- [ ] Change the package settings page content width to match with the other pages, so it has an empty sidebar like the others.
- [ ] Remove the package visibility settings section from the second step “set path to habitat plan file” of “connecting a plan file” task flow.

**Comment by chefsalim** [comment]
@apriofrost The mockup looks good to me!  

Splitting up the package connection to be per target platform is a good direction as well.  We could do that design as part of this task, or perhaps split it up into a separate work item (I believe there is already an issue to redesign the package connection)

**Comment by apriofrost** [comment]
Note: this issue will be addressed in another redesign effort for the "connecting plan files" flow. The mockup and tasks showed above are likely to change.

**Comment by apriofrost** [comment]
This issue will be addressed in #1189

**Comment by krasnow** [comment]
Fixed in #1189
